core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/PathElement2d.java

```diff
@@ -36,935 +36,940 @@
  */
 public abstract class PathElement2d implements PathElement2afp {
```

Comment:
Currently, this PR is compiling but is it no more sync with the master, due to the fact that your PR is forking from a too old commit in the master.
For synchronizing, you need to:
* synchronize your local code of this PR: `git pull`
* fix any conflict that is occuring after the merging process.
* commit the changes.
* push the sync code of your PR

This process is common to all Git PR, either at their first push and at updates.